docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -191,10 +219,6 @@ children:
     url: Boot-and-Mount-a-LoopBack-3-application.html
     output: 'web, pdf'
 
-- title: 'Calling other APIs'
-  url: Calling-other-APIs-and-web-services.html
-  output: 'web, pdf'
-
```

Comment:
There is an extra empty line at https://github.com/strongloop/loopback-next/blob/b7cd2c2bb15b132ca3899d5965519370e7c71a30/docs/site/sidebars/lb4_sidebar.yml#L22